### PR TITLE
Enable BMO_SETUP on kuttl tests

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl

File: ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml [hunk 1]
```diff
@@ -45,10 +45,17 @@ images:
   dockerfile_literal: |
     FROM openstack-operator-ci-build-deploy
     ARG KUTTL_VERSION=v0.15.0
+    RUN curl -L -o /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
     RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
-    RUN chmod +x ./kubectl-kuttl
+    RUN chmod +x ./kubectl-kuttl /usr/local/bin/yq
     RUN mv ./kubectl-kuttl /usr/local/bin
-    RUN yum install -y skopeo
+    ENV BMO_SETUP=true
```
Comment (fao89) [inline ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml:52]: force bmo_setup: https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L2111-L2114

Comment (rabi) [inline ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml:52]: Not sure what we're trying here ,prow jobs use compact cluster and not crc. BMO should be already installed and running there...

Comment (fao89) [reply ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml:52]: I didn't know that

**Pull request closed without merging**